test/Sentry.Tests/Protocol/SentryEventTests.SerializeObject_AllPropertiesSetToNonDefault_SerializesValidObject.verified.txt

```diff
@@ -1,93 +1,92 @@
-{
-  breadcrumbs: [
-    {
-      message: crumb,
-      timestamp: Date_MaxValue
-    },
-    {
-      category: category,
-      data: {
-        data-key: data-value
-      },
-      level: warning,
-      message: message,
-      timestamp: Date_MaxValue,
-      type: type
-    }
-  ],
-  contexts: {
-    .NET Framework: {
-      .NET Framework: "v2.0.50727", "v3.0", "v3.5",
-      .NET Framework Client: "v4.8", "v4.0.0.0",
-      .NET Framework Full: "v4.8"
-    },
-    context_key: context_value
```

Comment:
@SimonCropp - FYI, I couldn't get the tests to all pass until I removed the extra `context_key` row.  No matter what I did, the sort-order was different between framework versions.  .NET 6 would sort `context_key` first, but .NET Core 3.1 and .NET 4.8 would sort `.NET Framework` first.

I think this is the same issue with Verify I was mentioning with my OSS projects.  The dictionary sort is likely using `StringComparer.InvariantCulture`, where it should probably be using `StringComparer.Ordinal`. I'm not sure if that's in Verify's code, or in a dependency.

Reply:
it is in verify. can u share a test that repos it?

Reply:
A bit more testing, and I believe the problem isn't with Verify but with how we serialize the `Contexts` object in Sentry.  Fixed with #2031.